sdk/identity/azure-identity/azure/identity/_credentials/default.py

```diff
@@ -91,6 +95,8 @@ def __init__(self, **kwargs):
             except Exception as ex:  # pylint:disable=broad-except
                 # transitive dependency pywin32 doesn't support 3.8 (https://github.com/mhammond/pywin32/issues/1431)
                 _LOGGER.info("Shared token cache is unavailable: '%s'", ex)
+        if not exclude_visual_studio_code_credential:
+            credentials.append(VSCodeCredential())
```

Comment:
We should also document this in the docstring's list. It would be the new "4. The user currently signed in to Visual Studio Code."